modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleSettingLocalServiceImpl.java

```diff
@@ -53,6 +60,155 @@ public ObjectValidationRuleSetting addObjectValidationRuleSetting(
+			long objectValidationRuleId, String objectValidationRuleSettingName,
+			List<ObjectValidationRuleSetting>
+				updatedObjectValidationRuleSettings,
+			long userId)
+		throws PortalException {
+
+		for (ObjectValidationRuleSetting keyObjectValidationRuleSetting :
+				keyObjectValidationRuleSettings) {
+
+			ObjectValidationRuleSetting
+				serviceBuilderObjectValidationRuleSetting =
+					objectValidationRuleSettingPersistence.fetchByOVRI_N_V(
+						objectValidationRuleId, objectValidationRuleSettingName,
+						keyObjectValidationRuleSetting.getValue());
+
+			if (serviceBuilderObjectValidationRuleSetting != null) {
+				serviceBuilderObjectValidationRuleSetting.setName(
+					keyObjectValidationRuleSetting.getName());
+				serviceBuilderObjectValidationRuleSetting.setValue(
+					keyObjectValidationRuleSetting.getValue());
+
+				updatedObjectValidationRuleSettings.add(
+					updateObjectValidationRuleSetting(
+						serviceBuilderObjectValidationRuleSetting));
+
+				continue;
+			}
+
+			updatedObjectValidationRuleSettings.add(
+				addObjectValidationRuleSetting(
+					userId, objectValidationRuleId,
+					keyObjectValidationRuleSetting.getName(),
+					keyObjectValidationRuleSetting.getValue()));
+		}
+	}
+
+	private void _deleteOldObjectValidationRuleSettings(
+			long objectValidationRuleId, String objectValidationRuleSettingName,
+			List<ObjectValidationRuleSetting> objectValidationRuleSettings,
+			List<ObjectValidationRuleSetting> oldObjectValidationRuleSettings)
+		throws PortalException {
+
+		Set<String> deleteOldObjectObjectValidationRuleSettingValues =
+			SetUtil.asymmetricDifference(
+				TransformUtil.transform(
+					oldObjectValidationRuleSettings,
+					ObjectValidationRuleSetting::getValue),
+				TransformUtil.transform(
+					objectValidationRuleSettings,
+					ObjectValidationRuleSetting::getValue));
+
+		for (String deleteOldObjectObjectValidationRuleSettingValue :
+				deleteOldObjectObjectValidationRuleSettingValues) {
+
+			ObjectValidationRuleSetting
+				serviceBuilderObjectValidationRuleSetting =
+					objectValidationRuleSettingPersistence.findByOVRI_N_V(
+						objectValidationRuleId, objectValidationRuleSettingName,
+						deleteOldObjectObjectValidationRuleSettingValue);
+
+			deleteObjectValidationRuleSetting(
+				serviceBuilderObjectValidationRuleSetting.
+					getObjectValidationRuleSettingId());
+		}
+	}
+
+	private void _getDeleteObjectValidationRuleSettings(
+		List<ObjectValidationRuleSetting> keyObjectValidationRuleSettings,
+		List<ObjectValidationRuleSetting> objectValidationRuleSettings,
+		List<ObjectValidationRuleSetting> outputObjectValidationRuleSettings) {
+
+		for (ObjectValidationRuleSetting objectValidationRuleSetting :
+				objectValidationRuleSettings) {
+
+			if (objectValidationRuleSetting.compareName(
+					ObjectValidationRuleSettingConstants.
+						NAME_KEY_OBJECT_FIELD_ID)) {
+
+				keyObjectValidationRuleSettings.add(
+					objectValidationRuleSetting);
+			}
+			else {
```

Comment:
Here can we use the `continue` statement instead `else` to keep the same pattern ?

Reply:
Sure!
Thanks for the tip!